docs/static/settings-file.asciidoc

```diff
@@ -130,6 +130,22 @@ guaranteed, but you save the processing cost of preserving order.
 
 | `auto`
 
+| `pipeline.ecs_compatibility`
+a|
+Sets the pipeline's default value for `ecs_compatibility`, a setting that is available to plugins that implement an ECS compatibility mode for use with the Elastic Common Schema.
+Possible values are:
+
+* `disabled`
+* `v1`
+* `v8`
+
+This option allows the <<{logstash-ref}/ecs-ls.htm,early opt-in (or preemptive opt-out) of ECS compatibility>> modes in plugins,
```

Comment:
```suggestion
This option allows the <<ecs-ls,early opt-in (or preemptive opt-out) of ECS compatibility>> modes in plugins,
```

Reply:
This notation will work better for what you're trying to do (since we're linking within LS and no plugins are involved).

Reply:
🙇🏼